Project.toml

```diff
@@ -10,7 +10,6 @@ ExaTron = "28b18bf8-76f9-41ea-81fa-0f922810b349"
 FileIO = "5789e2e9-d7fb-5bc7-8068-2c6fae9b9549"
 Ipopt = "b6b21f68-93f8-5de0-b562-5493be1d77c9"
 JLD2 = "033835bb-8acc-5ee8-8aae-3f567f8a3819"
-JuMP = "4076af6c-e467-56ae-b986-b466b2749572"
 LazyArtifacts = "4af54fe1-eca0-43a8-85a7-787d91b784e3"
```

Comment:
It may be sufficient to have this for testing. So we can remove this too.

Reply:
Done.